warehouse/packaging/tasks.py

```diff
@@ -54,6 +60,67 @@ def sync_file_to_cache(request, file_id):
         file.cached = True
 
 
+@tasks.task(ignore_result=True, acks_late=True)
+def backfill_metadata(request):
+    metrics = request.find_service(IMetricsService, context=None)
+    is_pypi = request.registry.settings.get("warehouse.domain") == "pypi.org"
+    subdomain = "files" if is_pypi else "test-files"
+    base_url = f"https://{subdomain}.pythonhosted.org/packages"
```

Comment:
Should be able to use config "files.url"?

Reply:
Nice catch